src/main/java/io/github/moulberry/notenoughupdates/profileviewer/PlayerStats.java

```diff
@@ -705,6 +705,16 @@ public static int getMagicalPower(JsonObject inventoryInfo) {
 						break;
 				}
 			}
+			if (entry.getKey().equals("ABICASE")) {
+				if (profileInfo.has("nether_island_player_data") &&
+					profileInfo.get("nether_island_player_data").getAsJsonObject().has("abiphone") && profileInfo.get(
+					"nether_island_player_data").getAsJsonObject().get("abiphone").getAsJsonObject().has("active_contacts")) { // BatChest
+					int contact =
+						profileInfo.get("nether_island_player_data").getAsJsonObject().get("abiphone").getAsJsonObject().get(
+							"active_contacts").getAsJsonArray().size();
+					powderAmount += Math.floor(contact / 2);
```

Comment:
why is it called powder amount tho

Reply:
great question @Lulonaut

Reply:
nah this sounds like something hypixel would do, im sure its just because of the api

Reply:
> nah this sounds like something hypixel would do, im sure its just because of the api

Wat

Reply:
Great question